styles/_mixins.scss

```diff
@@ -22,3 +22,25 @@
   pointer-events: none;
   opacity: 0.2;
 }
+
+// todo: refactor this style once we have designs
+@mixin processingOverlay {
+  position: relative;
```

Comment:
Though if this does have height 100%, the content message may not always be visible, depending on where the scrollbar is.